### PR TITLE
[W.I.P.]Feature/reintroduce prettier to pipeline

### PR DESCRIPTION
**Pull request by raphaellueckl**
Changes proposed in this pull request:

 - Reintroduce prettier to pipeline
 - Probably get rid of some eslint rules that trigger because of that

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -39,12 +41,19 @@ const generatePlugins = type => {
...
+            'stylesheets'
+          ),
+        ],
         data: globals,
       },
-      processor: css => postcss([autoprefixer, stripFontFace])
-        .process(css)
-        .then(result => result.css),
+      processor: css =>
+        postcss([autoprefixer, stripFontFace])
```
Comment (LucaMele) [inline rollup.config.js]: diese schreibweise gefällt mir aber gar nicht

File: rollup.config.js [hunk 2]
```diff
@@ -89,20 +90,23 @@ const generatePlugins = type => {
...
 const generateOutputConfig = type => entry => ({
   input: entry,
-  external: type !== LIB ? undefined : [
-    'lit-element',
-    'lit-html/directives/class-map',
-    'lit-html/directives/repeat',
-    '@skatejs/val'
-  ],
+  external:
+    type !== LIB
```
Comment (LucaMele) [inline rollup.config.js]: dies ist weniger leserlich

File: src/components/10-atoms/button/ui.test.js [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
 });
 
 test('should style button default css (test axa blue bg color)', async t => {
-  const $axaButtonShadow = await Selector(() => document.querySelector('axa-button').shadowRoot);
+  const $axaButtonShadow = await Selector(
+    () => document.querySelector('axa-button').shadowRoot
+  );
   const $axaButtonShadowEl = await $axaButtonShadow.find(BUTTON_CLASS);
-  await t.expect(await $axaButtonShadowEl.getStyleProperty('background-color')).eql('rgb(0, 0, 143)');
+  await t
```
Comment (LucaMele) [inline src/components/10-atoms/button/ui.test.js]: dies ist nicht leserlich

File: src/components/20-molecules/datepicker/test.accessor.js [hunk 4]
```diff
@@ -99,36 +113,48 @@ class DatePickerAccessor {
...
-      .shadowRoot.querySelector('axa-dropdown[class*="js-datepicker__dropdown-month"]')
+      .shadowRoot.querySelector(
+        'axa-dropdown[class*="js-datepicker__dropdown-month"]'
+      )
       .shadowRoot.querySelector('button[data-value="1"]')
   );
 
   selectMonthDropdownSelector = Selector(id =>
-    document.querySelector(`axa-datepicker[data-test-id="${id}"]`).shadowRoot.querySelector('axa-dropdown')
+    document
```
Comment (LucaMele) [inline src/components/20-molecules/datepicker/test.accessor.js]: dies ist nicht leserlich

File: src/components/20-molecules/datepicker/test.accessor.js [hunk 5]
```diff
@@ -31,7 +31,9 @@ class DatePickerAccessor {
   async assertYear(year) {
     const yearDropdown = await Selector(this.yearDropdownSelector(this.id));
     await this.t.expect(yearDropdown.exists).ok();
-    await this.t.expect(yearDropdown.getAttribute('items')).contains(`"value":"${year}"`);
+    await this.t
```
Comment (LucaMele) [inline src/components/20-molecules/datepicker/test.accessor.js]: this is not readable

File: src/components/20-molecules/datepicker/test.accessor.js [hunk 6]
```diff
@@ -72,21 +78,29 @@ class DatePickerAccessor {
...
   submitButtonSelector = Selector(id =>
-    document.querySelector(`axa-datepicker[data-test-id="${id}"]`).shadowRoot.querySelector('axa-button[class*="js-datepicker__button-ok"]')
+    document
+      .querySelector(`axa-datepicker[data-test-id="${id}"]`)
+      .shadowRoot.querySelector('axa-button[class*="js-datepicker__button-ok"]')
   );
 
   datepickerInputFieldSelector = Selector(id =>
-    document.querySelector(`axa-datepicker[data-test-id="${id}"]`).shadowRoot.querySelector('input[class*="js-datepicker__input"]')
+    document
```
Comment (LucaMele) [inline src/components/20-molecules/datepicker/test.accessor.js]: totally not readable

File: src/components/20-molecules/datepicker/test.accessor.js [hunk 7]
```diff
@@ -99,36 +113,48 @@ class DatePickerAccessor {
   });
 
   openCalendarSelector = Selector(id =>
-    document.querySelector(`axa-datepicker[data-test-id="${id}"]`).shadowRoot.querySelector('button[class*="m-datepicker__input-button"]')
+    document
```
Comment (LucaMele) [inline src/components/20-molecules/datepicker/test.accessor.js]: dtto

**Comment by LucaMele** [comment]
Brakes eslint rules

Bildschirmfoto 2019-04-18 um 09 18 26

**Comment by raphaellueckl** [comment]
This PR will not be merged, since we already have conflicts and the teams needs to decide again, if we want prettier.

**Pull request closed without merging**